specifications/tcp/tcp.tla

```diff
@@ -0,0 +1,330 @@
+------------------------------- MODULE tcp ----------------------------------
+(* https://datatracker.ietf.org/doc/html/rfc9293 *)
```

Comment:
I think it would make sense to explain a bit more in detail what exactly is modeled. The RFC talks a lot about sequence numbers etc., but these aspects are abstracted from here (which makes perfect sense).